.travis.yml

```diff
@@ -1,8 +1,6 @@
 # This file was generated on 2020-09-28T21:53:54+02:00 from the rspec-dev repo.
 # DO NOT modify it by hand as your changes will get lost the next time it is generated.
 
-# In order to install old Rubies, we need to use old Ubuntu distibution.
-dist: trusty
```

Comment:
This is the line where it got JRuby 9.2.13.0 stuck. I have no idea why but assuming this is only needed for Ruby 1.8 and this merge request is not aiming for Ruby 1.8 that should be fine.

I want to get this ready except anything related to Ruby 1.8. I know we wouldn't edit this file directly but this is a demonstration that everything beside that should work.